bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Widget.java

```diff
@@ -52,7 +52,19 @@
  */
 public abstract class Widget {
 
-	private int zoom;
+	/**
+	 * the native zoom of the monitor in percent
+	 * (Warning: This field is platform dependent)
+	 * <p>
+	 * <b>IMPORTANT:</b> This field is <em>not</em> part of the SWT
+	 * public API. It is marked public only so that it can be shared
+	 * within the packages provided by SWT. It is not available on all
+	 * platforms and should never be accessed from application code.
+	 * </p>
+	 *
+	 * @noreference This field is not intended to be referenced by clients.
+	 */
+	protected int nativeZoom;
```

Comment:
Can't this be package protected (for the scope of this PR) and increased in visibility once required?